### PR TITLE
Add parcel support

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: packages/parcel-transformer/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Transformer } from '@parcel/plugin';
+import { compile, processVanillaFile } from '@vanilla-extract/integration';
+
+export default new Transformer({
+  async transform({ asset, options }) {
+    const identOption = options.mode === 'development' ? 'debug' : 'short';
```
Comment (markdalgleish) [inline packages/parcel-transformer/src/index.ts:6]: Should consumers also be able to explicitly set this via an `identifiers` option?

File: fixtures/features/package.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fixtures/features",
   "version": "0.0.13",
-  "main": "src/index.ts",
```
Comment (markdalgleish) [inline fixtures/features/package.json:4]: Is this because Parcel treats the HTML file as the entry point?

Comment (mattcompiles) [reply fixtures/features/package.json:4]: Yeah, and it complains about this in that case.

File: package.json [hunk 3]
```diff
@@ -37,6 +37,7 @@
     "@changesets/changelog-github": "^0.4.0",
     "@changesets/cli": "^2.17.0",
     "@manypkg/cli": "^0.19.1",
+    "@parcel/packager-ts": "2.7.0",
```
Comment (markdalgleish) [inline package.json:40]: I don't see this referenced anywhere else. Is this meant to be here?

Comment (mattcompiles) [reply package.json:40]: It's part of parcels auto install thing. I'll double check though.

**Comment by tanishqkancharla** [comment]
What else needs to be done here for merge? I would like to use this.

**Comment by kasperstorgaard** [comment]
I would also love to see this land ☝️

**Review by askoufis — COMMENTED** [review]
LGTM, needs a changeset though

File: site/docs/integrations/parcel.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: parcel
+parent: integrations
+---
+
+# parcel
+
+A parcel transformer for integrating vanilla-extract with [parcel](https://parceljs.org/).
```
Comment (mrm007) [inline site/docs/integrations/parcel.md:8]:
```suggestion
A parcel transformer for integrating vanilla-extract with [Parcel](https://parceljs.org/).
```

File: site/docs/integrations/parcel.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Installation
+
+```bash
+npm install --save-dev @vanilla-extract/parcel-transformer
+```
+
+## Setup
+
+Add the transformer to your parcel configuration.
```
Comment (mrm007) [inline site/docs/integrations/parcel.md:18]:
```suggestion
Add the transformer to your Parcel configuration.
```

File: site/docs/integrations/parcel.md (hunk 4 above)
Comment (mrm007) [inline site/docs/integrations/parcel.md:8]:
```suggestion
A Parcel transformer for integrating vanilla-extract with [parcel](https://parceljs.org/).
```

**Review by askoufis — APPROVED** [review]

**Comment by askoufis** [comment]
> @askoufis Did I miss it, or has this not been released yet?

Because it was a new package and there wasn't a changeset, there's no release entry on the repo release page, but it did get released, it's on [here](https://www.npmjs.com/package/@vanilla-extract/parcel-transformer).

**Pull request merged**